code/modules/reagents/reagents/medical.dm

```diff
@@ -1183,3 +1183,87 @@
 			continue
 		limb_to_unfix.fracture()
 		break
+
+
+/datum/reagent/medicine/research
```

Comment:
Is having a parent type purely for organisation, or are you planning something else with it?

Reply:
Both, I might do something with them. I'm considering just leaving them as reagent/research rather than medicine/research, but I'm still thinking it through.